src/components/clusterDetail/ResetClusterModal.tsx

```diff
@@ -67,28 +80,58 @@ const ResetClusterModal: React.FC<ResetClusterModalProps> = ({ onClose, isOpen,
           This will reset the installation and return to cluster configuration. Some hosts may need
           to be re-registered by rebooting into the Discovery ISO.
```

Comment:
```suggestion
          This will reset the installation and return to the cluster configuration. Some hosts may need
          to be re-registered by rebooting into the Discovery ISO.
```